Makefile

```diff
@@ -100,6 +100,36 @@ docker-test: docker-build
 
 .PHONY: docker-build docker-build-release docker-run docker-test
 
+# --- Deployment -------------------------------------------------------------------
```

Comment:
I think the commit message is wrongly focused. This commit has little to do with adding a sealed secret. It is about adding the deployment machinery, which includes generating a random sealed secret, but also the overlay too.

And as I said in the later commit, this should just include the diff and undeploy targets too.

Reply:
ok.